app/assets/javascripts/discourse/app/templates/components/sidebar/section-link.hbs

```diff
@@ -1,63 +1,71 @@
 <div class="sidebar-section-link-wrapper">
-  <Sidebar::SectionLinkTo
-    @class={{concat @class (concat " sidebar-section-link sidebar-section-link-" @linkName)}}
-    @route={{@route}}
-    @query={{@query}}
-    @models={{if @model (array @model) (if @models @models (array))}}
-    @current-when={{@currentWhen}}
-    @title={{@title}}
-  >
-    {{#if @prefixValue }}
-      <span class="sidebar-section-link-prefix {{@prefixType}} {{@prefixCSSClass}}" style={{this.prefixCSS}}>
-        {{#if (eq @prefixType "image")}}
-          <img src={{@prefixValue}} class="prefix-image">
-        {{/if}}
+  {{#if @href}}
+    <a href={{@href}} rel="noopener noreferrer" target="_blank" class={{this.classNames}} title={{@title}}>
```

Comment:
I wonder if this will be a component at some point? Not sure if there would be a benefit to just curious.

Reply:
I'm not sure if there is a benefit of extracting a `a` tag into a component unless we have certain behaviour that needs to rely on computed properties or modifiers.